src/test/librados_test_stub/MockTestMemIoCtxImpl.h

```diff
@@ -106,6 +106,7 @@ class MockTestMemIoCtxImpl : public TestMemIoCtxImpl {
                      std::map<uint64_t, uint64_t> *m, bufferlist *bl){
      return TestMemIoCtxImpl::sparse_read(oid, off, len, m, bl);
   }
+  using TestMemIoCtxImpl::sparse_read;
```

Comment:
The warning is that the ```size_t len``` parameter doesn't match -- the param type needs to be fixed and then you can removed this ```using```.